test/functional/test_framework/test_shell.py

```diff
@@ -7,6 +7,14 @@
 from test_framework.test_framework import BitcoinTestFramework
 
 
+# BitcoinTestFramework instances are supposed to be constructed with the path
+# of the calling test in order to find shared data like configuration and the
+# cache. Since TestShell is meant for interactive use, there is no concrete
```

Comment:
Now that it's moved out of `TestShell` this (slightly awkward) comment should likely be updated.